### DISCUSSION TITLE
Phone value from google oauth

### DISCUSSION THREAD

**Discussion by v-romanyuk** [supabase/supabase#26227] [Questions]
Hello! I'm trying to obtain a user's phone number during the Google OAuth login process. I've included the scope https://www.googleapis.com/auth/user.phonenumbers.read to request access to the user's phone number. Additionally, I have enabled phone number sharing in my Google account preferences and on the Google consent screen. Despite these settings, the phone field remains empty in the Supabase session. What could be causing this issue, and how can I successfully retrieve the phone number?

**Comment by GaryAustin1** [comment]
You might check more on Google's side of it.
Found this which says may not be possible.  https://stackoverflow.com/questions/65339087/google-sign-in-for-websites-read-users-phone-number
And a link that seems to show scopes (phone not there) https://www.googleapis.com/discovery/v1/apis/oauth2/v2/rest?fields=auth(oauth2(scopes))

But I'm just giving  you some info to research as I don't know for a fact you can or can't.

**Reply by v-romanyuk** [reply]
Thanks for your reply. Will take a look
